.typos.toml

```diff
@@ -12,7 +12,11 @@ extend-exclude = [
   "node-metrics/src/api/node_validator/v0/example_prometheus_metrics_output.txt",
   "crates/hotshot/orchestrator/run-config.toml",
   "crates/hotshot/macros/src/lib.rs",
+  "crates/hotshot/types/src/light_client.rs",
   "staking-cli/tests/cli.rs",
   "sdks/go/types/common/consts.go",
   "data/insta_snapshots",
 ]
+
+[default.extend-words]
+Forgetten = "Forgetten"
```

Comment:
Lol, replace all with forgotten?

Reply:
This is so funny.😂 you successfully engraved typos into an immutable ledger, bytes level cemented.

Irrefutable personal footprint.